src/app/view/components/_common/dot-textarea-content/dot-textarea-content.component.ts

```diff
@@ -91,6 +106,10 @@ export class DotTextareaContentComponent implements OnInit, ControlValueAccessor
         $event.stopPropagation();
     }
 
+    onInit(editor) {
```

Comment:
```suggestion
    onInit(editor: any): void {
```

Needs doc. Remember "Document This" extension for all the public method.